### ISSUE TITLE
Docs: update logos in images

### ISSUE THREAD

**Issue by tpdorsey** [PrefectHQ/prefect#5308]
## Description
Some images in the documentation have the old Prefect logo. These should be updated to the new blue logo.

To do issue. List of specific images TK.

## Environment

Optionally run `prefect diagnostics` from the command line and paste the information here. -->

**Comment by harshlancer** [comment]
@tpdorsey If any one is not assigned can I do this?

**Comment by harshlancer** [comment]
Yes  https://docs.prefect.io/api/latest/, doesn't have gradient logos.

So what should I do next? 

Please don't mind my naiveness I'm just starting in open-source communtity.

**Comment by tpdorsey** [comment]
No worries. This is what I'd do:
1. Update the svg file at .vuepress/public/assets to the new gradient logo from the link above.
2. Change the URL ref for the image on those pages to point at the relative URL of the new image (/assets/prefect-logo-full-gradient.svg)
3. For the navbar logo, you should just have to update the file /assets/logomark-color.png.
4. Test locally to make sure the images are loading correctly, then push your PR.

If you haven't contributed yet, take a look through the development guide. It's pretty helpful for getting set up correctly to make contributions: https://docs.prefect.io/core/development/overview.html

Let me know if you have any further questions.

**Comment by harshlancer** [comment]
I've cloned the project but I'm not able to find .vuepress/public/assets I've searched about it and this is what I've found 

https://vuepress.vuejs.org/guide/getting-started.html#manual-installation

if this is correct I can start @tpdorsey

**Comment by tpdorsey** [comment]
@harshlancer You don't need to install Vuepress to make these changes. 

The full path in the repo is /prefect/docs/.vuepress/public/assets. If you're looking in the filesystem, you may need to turn on displaying hidden files. On macos that's command-shift-. (command-shift-period). If you're in a terminal, you'll need `ls -a` to see hidden files and directories.

To build and preview the docs you will need to follow the [dev setup instructions here](https://docs.prefect.io/core/development/overview.html#clone-prefect) because you need to have Prefect installed to build the docs. To build and serve the docs, you'll also need NodeJS 14 (I have 14.8.1). 

I'll add some step-by-step notes about getting set up for a PR submission at the end should you need them. 

Just to make sure the [steps to get set up for a docs contribution](https://docs.prefect.io/core/development/overview.html#submit-your-code) are clear:

1. Make your own fork of the PrefectHQ/prefect repo.
2. Clone _your fork_ of the prefect repo locally.
3. [Install Prefect from the clone](https://docs.prefect.io/core/development/overview.html#clone-prefect)
4. Create a git branch for your changes. For docs changes, I like to name my branches starting with "docs-", so something like "docs-gradientlogos" would be good, but certainly not a requirement.
5. Make your changes in this branch.
6. Please preview your changes locally before submitting them to make sure everything looks okay.
7. When you're ready, push your change branch up to your fork: `git push origin <branch>`. Then go up to Github and it will prompt you to create a PR.

Feel free to ask me any questions you may have about this process, if you haven't done it before.
